src/spikeinterface/extractors/phykilosortextractors.py

```diff
@@ -197,18 +201,26 @@ class PhySortingExtractor(BasePhyKilosortSortingExtractor):
         Path to the output Phy folder (containing the params.py).
     exclude_cluster_groups: list or str, optional
         Cluster groups to exclude (e.g. "noise" or ["noise", "mua"]).
```

Comment:
yes! let me do it since I'm at it :)

Reply:
@zm711 should be good to go. Also added typing

Reply:
@alejoe91 looks good to me. Also with the new typing (although it is a bit annoying/also very Python) that you can also do `list[str]` and use the built in type rather than the typing `List`. That's not a suggestion, it's more of a rant that sometimes a little less flexibility makes it easier for programmers to be consistent. :)

Reply:
It fails with python 3.8 though! So as long as we keep support for 3.8 we should stick to this IMO :)

Reply:
Then one general question does `__futures__` not include in that? I haven't actually tried that but `|` is only 3.10+ so I thought futures would technically handle both cases? But I could be wrong I have never read what all __futures__ ports?

Reply:
good point. I'm sure @h-mayorquin  would know :)

Reply:
![image](https://github.com/SpikeInterface/spikeinterface/assets/17097257/977130ee-2561-45bf-b111-df4e82f18e72)

Seems like it works!

Reply:
Now that's the scientist: just do the test and prove and the hypothesis :)

Reply:
> Now that's the scientist: just do the test and prove the hypothesis :)

I wish science was that easy!!! :)

Reply:
Empiricism for the win!